src/DropdownStateMixin.js

```diff
@@ -56,7 +57,7 @@ const DropdownStateMixin = {
   },
 
   bindRootCloseHandlers() {
-    let doc = React.findDOMNode(this).ownerDocument;
+    let doc = domUtils.ownerDocument(React.findDOMNode(this));
```

Comment:
That was my first thought too, but decided against it because `domUtils` (currently) doesn't deal with anything React specific, just with the browser dom. If you think it's okay to introduce React there I can change it of course.